src/optics/lib/Convex/CardanoApi/Lenses.hs

```diff
@@ -88,7 +87,6 @@ module Convex.CardanoApi.Lenses (
 
   -- * Datums
   _TxOutDatumInline,
-  _TxOutDatumInTx,
   _TxOutDatumHash,
```

Comment:
Wasn't just `TxOutDatumInTx` been renamed to `TxOutSupplementalDatum`? Then, we
should have a prism for it.